graphql/playground.ts

```diff
@@ -2,17 +2,17 @@ import { ApolloServerPluginLandingPageGraphQLPlayground } from 'apollo-server-co
 import { isDev } from '../common/util/environment';
 
 const userHeader = { authorization: 'Bearer <somerandombearer>' };
-const adminHeader = { authorization: "Basic <btoa('admin' + password)>" };
+const adminHeader = { authorization: `Basic ${btoa('admin:' + process.env.ADMIN_AUTH_TOKEN)}` };
```

Comment:
Mmm can we at least exclude productive envs here? With this we might leak the admin token in the future

Reply:
The admin headers are only attached to dev environments, no?

Reply:
For now yes, but someone might accidentally change it in the future